src/Document.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Ipssi\Evaluation;
+
+class Document {
+
+    /*private $text;
+    private $image;
+    private $xImage;
+    private $yImage;
+    private $xText;
+    private $yText;*/
+
+    /*public function __construct(array $elementText, $elementImage)
+    {
+        $this->text = $text;
+        $this->image = $image;
+        $this->x = $x;
+        $this->y = $y;
+        $this->text = $elementText["TEXT"];
+        $this->xText = $elementText["X"];
+        $this->yText = $elementText["Y"];
+
+        $this->image = $elementImage["IMAGE"];
+        $this->xImage = $elementImage["X"];
+        $this->yImage = $elementImage["Y"];
+    }
+
+    public function afficheDocument(): string
+    {
+        return "==== DOCUMENT ====" . PHP_EOL .
+            "=== TEXT ===" . PHP_EOL .
+            $this->text . PHP_EOL .
+            $this->xText . PHP_EOL .
+            $this->yText . PHP_EOL .
+            "=== IMAGE ===" . PHP_EOL .
+            $this->image . PHP_EOL .
+            $this->xImage . PHP_EOL .
+            $this->yImage . PHP_EOL;
+    }*/
+
+
+    private $elementText;
+
+    private $elementImage;
+
+    private $elementFormeEtoile;
+    private $elementFormeNuage;
+
+    public function __construct(
+        $elementText, $elementTextX, $elementTextY, $elementTextCouleur,
```

Comment:
On procède plutôt comme suit : 
```suggestion
public function __construct(
       ElementText $text,
       ElementImage $image
   ) {
       $this->elementText = $text;
       $this->elementImage = $image;
   }
```
Et c'est à l'extérieur du constructeur que nous faisons les new Element.
```php

new Document(new ElementText('toto'), new ElementImage());
```
et l'idée était plus de pouvoir ajouter autant d'éléments que souhaité, votre proposition est trop restrictive.